tests/plenary/state/state_spec.lua

```diff
@@ -0,0 +1,144 @@
+local utils = require('orgmode.utils')
+local state = nil
```

Comment:
```suggestion
---@type OrgState
local state = nil
```

Reply:
Done. I also renamed all the `State` objects to `OrgState`.